schema/cassandra/cadence/versioned/v0.12/events_v2.cql

```diff
@@ -25,4 +25,17 @@ CREATE TABLE history_tree (
   PRIMARY KEY ((tree_id), row_type, branch_id )
 ) WITH COMPACTION = {
 'class': 'org.apache.cassandra.db.compaction.LeveledCompactionStrategy'
-};
\ No newline at end of file
+};
+
+CREATE TYPE history_branch_info (
+  branch_token                     blob,
+  next_event_id                    bigint,
+  last_first_event_id              bigint,
+  history_size                     bigint,
+);
+
+--- using eventsV2
+ALTER TYPE workflow_execution ADD  event_store_version             int;
```

Comment:
Do we really need this?  Can we rely on history_tree_id being set as a mechanism to check if history is stored as v1 vs v2?

Reply:
I am going to remove that tree_id as we discussed. But even with that tree_id, I personally prefer to make it more straightforward if it doesn't introduce any issue(storage/perf) than using nil/not nil. Also checking nil with gocql is not always safe.